.appveyor/build_libedit.bat

```diff
@@ -0,0 +1,59 @@
+@echo on
+setlocal
+
+call "%~dp0set_paths.bat"
+
+cd "%BUILDDIR%"
+
+if exist "%BINARIESDIR%\libedit" (
```

Comment:
If it is a CI platform, what are the chances that libedit, postgres and other such third party will already be present on the machine? If they are then why do we error out?